pkgs/applications/editors/emacs/elisp-packages/manual-packages/emacs-conflict/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, fetchFromGitHub
+, trivialBuild
+, emacs
+}:
+
+trivialBuild {
+  pname = "emacs-conflict";
+  version = "9f236b93930f3ceb4cb0258cf935c99599191de3";
```

Comment:
Could you change it according the doc here: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming